frontends/web/src/components/guide/guide.tsx

```diff
@@ -22,7 +22,8 @@ import { CloseXWhite } from '@/components/icon';
 import { AppContext } from '@/contexts/AppContext';
 import { Button } from '@/components/forms';
 import style from './guide.module.css';
-
+import { getFeedbackLink } from '@/routes/settings/components/about/feedback-link-setting';
+import { getSupportLink } from '@/routes/settings/components/about/support-link-setting';
```

Comment:
Ideally routes import from components, but components don't import from routes and are standalone (we probably don't do that consistently across the codebase).

Could you change that both components (FeedbackLink and Guide) import from the same module, maybe from url_constants.ts or similar?